src/app/(UserContent)/layout.tsx

```diff
@@ -1,8 +1,8 @@
-import UserContentNavigation from "@/components/wrappers/UserContentNavigation";
+import UserContentNavigation from '@/components/wrappers/UserContentNavigation';
 
 export default function UserContentLayout({ children }: { children: React.ReactNode }) {
```

Comment:
There is a type in React called `PropsWithChildren`, and you can write it instead of `{ children: React.ReactNode }`

Reply:
nice note